e2e-cypress/files/tests/acceptance/acceptance.spec.ts

```diff
@@ -1,5 +1,32 @@
+function printEvidenceForPageElement
+  (testName: string, testStep: number, selector: string, description: string) {
+  if (!selector) {
+    throw new Error('selector must not NOT be undefined');
```

Comment:
I guess it should be ``'selector must be defined'`` or  ``'selector must not be undefined'``